### ISSUE TITLE
Autoplay is bugged out in the latest version (uYouEnhanced_19.06.2_3.0.2.ipa)

### ISSUE THREAD

**Issue by justinjohn0306** [arichornlover/uYouEnhanced#343]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Have you read the FAQ?

- [X] Yes, I did read the FAQ

### iOS/iPadOS version

17.3.1

### Device model

iPhone 15 Pro Max

### Sideload tool

Esign

### Tweak version

uYouEnhanced_19.06.2_3.0.2.ipa

### Describe the bug

The autoplay option is bugged out

### Steps to reproduce the issue

1. Download uYouEnhanced_19.06.2_3.0.2.ipa
2. Install it
3. Run it

### Are you using the newest version of uYouPlus? If not, why?

✅ Yes, I'm using the latest version of uYouPlus right now

### Does the issue happen with the official YouTube from AppStore?

❌ No, I can't reproduce this issue in the official YouTube from AppStore

**Comment by revfast** [comment]
Yes I can confirm autoplay doesn’t work. But it works fine on the app installed via AppStore

**Comment by thexYeti** [comment]
same for me, did you find a fix?

**Comment by revfast** [comment]
> same for me, did you find a fix?

"I literally just fixed this on my end. I had to go to uyou settings, then video player and turn on “no suggested videos at the end” and that fixed it for me." User Twain10 on Reddit found this and it worked for me, I'm on qnblackhat latest version that seems to have the same issue

**Comment by longopy** [comment]
> > same for me, did you find a fix?
> 
> "I literally just fixed this on my end. I had to go to uyou settings, then video player and turn on “no suggested videos at the end” and that fixed it for me." User Twain10 on Reddit found this and it worked for me, I'm on qnblackhat latest version that seems to have the same issue

I tried it and it doesn't seem to work for me.

**Comment by longopy** [comment]
It still doesn't work for me. When a video ends, playback stops. It does not continue playing another video, as it did in previous versions

**Comment by longopy** [comment]
> > It still doesn't work for me. When a video ends, playback stops. It does not continue playing another video, as it did in previous versions 
> 
> Have you tried only enabling the option only in uYou and leaving it off in uYouEnhanced settings?

 Thank you very much! Fixed by disabling uYouEnhanced settings. They seem to conflict.
